templates/summary.html

```diff
@@ -8,13 +8,20 @@
 
 {% block form_content %}
   {% if content.summary.summary_type == 'Summary' %}
-    <h1>{{ _("Check your answers and submit") }}</h1>
+    <h1>{{ content.title }}</h1>
+
+    {% if content.warning %}
+        {% call onsPanel({
+            "type": "warn"
+          }) %}
+          <p data-qa="warning">{{ content.warning }}</p>
+        {% endcall %}
+    {% endif %}
 
     {% call onsPanel({
       "classes": "print__hidden u-mb-l"
     }) %}
-      <strong>{{ _("Please submit this survey to complete it") }}</strong>
-      <p>{{ _("Check your answers before submitting") }}</p>
```

Comment:
I couldn't see on the card this being removed, was it requested by Donna?

Reply:
@LJBabbage confirmed with Donna that this text is no longer required